pkg/buildah/login.go

```diff
@@ -48,6 +54,8 @@ func (opts *loginReply) RegisterFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&opts.tlsVerify, "tls-verify", opts.getLogin, "require HTTPS and verify certificates when accessing the registry. TLS verification cannot be used when talking to an insecure registry.")
 	fs.BoolVar(&opts.getLogin, "get-login", opts.tlsVerify, "return the current login user for the registry")
 	fs.AddFlagSet(auth.GetLoginFlags(&opts.loginOpts))
+	// e.g sealos login --kubeconfig /root/.kube/config hub.sealos.io
+	fs.StringVarP(&opts.kubeconfig, "kubeconfig", "k", opts.kubeconfig, "Login to sealos registry: hub.sealos.io by kubeconfig")
```

Comment:
set a default value `~/.kube/config`

Reply:
Seems not necessary. If we downloaded kubeconfig from our cloud site, the file will be in any location.

There is only one statement that`~/.kube/config` exists, witch is users using `sealos run` to start and manage a cluster, witch at this statement, they are not likely do login or push images, but  just pull and run sealos images.